### PR TITLE
Added post endpoint of employee

### PR DESCRIPTION
**Pull request by chaitanya-chorawala**

### CODE REVIEW

File: GitCore/Controllers/EmployeeController.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GitCore.Controllers
+{
```
Comment (iDhruvi) [inline GitCore/Controllers/EmployeeController.cs:5]: hisdmis

**Review by iDhruvi — COMMENTED** [review]
Gooood

File: GitCore/Controllers/EmployeeController.cs (hunk 1 above)
Comment (chaitanya-chorawala) [reply GitCore/Controllers/EmployeeController.cs:5]: resolved

Comment (iDhruvi) [reply GitCore/Controllers/EmployeeController.cs:5]: very bad

**Pull request merged**